src/libpriv/rpmostree-bwrap.c

```diff
@@ -72,48 +222,69 @@ rpmostree_bwrap_base_argv_new_for_rootfs (int rootfs_fd, GError **error)
       else if (!S_ISLNK (stbuf.st_mode))
         continue;
 
-      g_ptr_array_add (bwrap_argv, g_strdup ("--symlink"));
-
-      path = g_strconcat ("usr/", subdir, NULL);
-      g_ptr_array_add (bwrap_argv, path);
+      srcpath = g_strconcat ("usr/", subdir, NULL);
+      destpath = g_strconcat ("/", subdir, NULL);
+      rpmostree_bwrap_append_bwrap_argv (ret, "--symlink", srcpath, destpath, NULL);
+    }
 
-      path = g_strconcat ("/", subdir, NULL);
-      g_ptr_array_add (bwrap_argv, path);
+  switch (mutable)
+    {
+    case RPMOSTREE_BWRAP_IMMUTABLE:
+      rpmostree_bwrap_append_bwrap_argv (ret, "--ro-bind", "usr", "/usr", NULL);
+      break;
+    case RPMOSTREE_BWRAP_MUTATE_ROFILES:
+      if (!setup_rofiles_usr (ret, error))
+        goto out;
+      /* Note intentional switch fallthrough */
```

Comment:
Hmm, I'm not following. I'm not suggesting we run `rofiles-fuse` in the container. We call `rofiles-fuse ./usr /tmp/xxxxxx` which will create the mountpoint at `/tmp/xxxxxx`, but then we still end up mapping (the unprotected) `./usr` to `/usr`, rather than (the protected) `/tmp/xxxxxx`. Doesn't that mean we're completely bypassing the `rofiles-fuse` mount when we shouldn't?